object-manipulation-1/src/set-value.js

```diff
@@ -0,0 +1,6 @@
+/* exported setValue */
+function setValue(object, key, value)
+{
```

Comment:
Please keep the opening curly brace for function definitions on the same line as the name and parameter list